### PR TITLE
Fix for sending notifications with priority 2

### PR DESCRIPTION
**Pull request by mrkooll**
To send notifications with priority 2 two additional parameters should be set (expire and retry).

This patch adds support for these parameters.

### CODE REVIEW

File: pushover.sh [hunk 1]
```diff
@@ -33,6 +33,8 @@ showHelp()
         echo "                                0 - normal priority"
         echo "                                1 - bypass the user's quiet hours"
         echo "                                2 - require confirmation from the user"
+        echo "  -E, --expire               Set expiration time for notifications with priority 2 (default 180)"
```
Comment (akusei) [inline pushover.sh]: Why use a capital E and R instead of just lowercase e and r, this would keep them in line with the other options

Comment (mrkooll) [reply pushover.sh]: I used uppercase to mark these non-required optional parameters with default values.
Can use lowercase instead.

Comment (akusei) [reply pushover.sh]: Is it a typical standard for CLI tools to make optional parameters uppercase?

Comment (akusei) [reply pushover.sh]: I tried to find some documentation on CLI options and best practices but only found this http://docopt.org/
I'd appreciate it if you could make those lowercase and I'll get this approved and merged, and thanks for the addition!

Comment (akusei) [reply pushover.sh]: I would also add 2 readonly variables at the top for EXPIRE_DEFAULT and RETRY_DEFAULT, then use those in place of your hard 180 and 30 both in the usage an in the check further down in the script

Comment (akusei) [reply pushover.sh]:
```suggestion
        echo "  -e, --expire               Set expiration time, in seconds, for notifications with priority 2 (default ${EXPIRE_DEFAULT})"
```

File: pushover.sh [hunk 2]
```diff
@@ -33,6 +33,8 @@ showHelp()
...
+		echo "  -R, --retry                Set retry period for notifications with priority 2 (default 30)"
```
Comment (akusei) [inline pushover.sh]:
```suggestion
		echo "  -r, --retry                Set retry period, in seconds, for notifications with priority 2 (default ${RETRY_DEFAULT})"
```

File: pushover.sh [hunk 3]
```diff
@@ -144,6 +146,16 @@ do
       shift
       ;;
 
+	-E|--expire)
```
Comment (akusei) [inline pushover.sh]:
```suggestion
	-e|--expire)
```

File: pushover.sh [hunk 4]
```diff
@@ -144,6 +146,16 @@ do
...
+		expire="${2:-}"
+		shift
+		;;
+
+	-R|--retry)
```
Comment (akusei) [inline pushover.sh]:
```suggestion
	-r|--retry)
```

File: pushover.sh [hunk 5]
```diff
@@ -156,6 +168,14 @@ do
   shift
 done
 
+if [ $priority -eq 2 ]; then
+	if [ -z "${expire:-}" ]; then
+		expire=181
```
Comment (akusei) [inline pushover.sh]:
```suggestion
		expire=${EXPIRE_DEFAULT}
```

File: pushover.sh [hunk 6]
```diff
@@ -156,6 +168,14 @@ do
...
+	fi
+	if [ -z "${retry:-}" ]; then
+		retry=30
```
Comment (akusei) [inline pushover.sh]:
```suggestion
		retry=${RETRY_DEFAULT}
```

**Review by akusei — CHANGES_REQUESTED** [review]
If you add the changes I made with the addition of the following, I'll get this merged

1) change line 1 from `#!/usr/bin/env sh` to `#!/usr/bin/env bash` or just rebase from master (this is a change already in master)
2) add the following lines at line 10
```bash
readonly EXPIRE_DEFAULT=180
readonly RETRY_DEFAULT=30
```

**Comment by mrkooll** [comment]
Updated according to requests.

**Review by akusei — APPROVED** [review]
Looks great, thanks!

**Pull request merged**